odoo/tools/xml_utils.py

```diff
@@ -83,3 +85,47 @@ def create_xml_node(parent_node, node_name, node_value=None):
     :returns (etree._Element):
     """
     return create_xml_node_chain(parent_node, [node_name], node_value)[0]
+
+
+def load_xsd_from_url(url, module_name, env):
+    """Load an xsd file from given url and save it as ir.attachment
+
+    :param url: url of xsd file (str)
+    :param module_name: name of calling module (str)
+    :param env: odoo.api.Environment
+    :returns: saved attachment in ir.attachment or None if error
+    """
+    _logger = logging.getLogger(__name__)
+
+    fname = url.split('/')[-1]
```

Comment:
should this end with `.xsd`? Can we add some sanity verifications to avoid we use this for something else?